lib/r10k/module/base.rb

```diff
@@ -66,6 +71,36 @@ def full_path
     path.to_s
   end
 
+  # Delete the spec dir unless @deploy_spec has been set to true or @spec_deletable is false
+  def delete_spec_dir
```

Comment:
I think we should rename this to something like `try_delete_spec_dir` and then the other one can be just `delete_spec_dir`. Makes it more obvious what the difference is between them.